webapp/src/routes/EditProfile/styles.js

```diff
@@ -0,0 +1,217 @@
+export default (theme) => ({
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    padding: theme.spacing(6, 1, 0, 1),
+    alignItems: 'center',
+    minHeight: 'calc(100vh - 60px)',
+    paddingTop: '60px',
+    [theme.breakpoints.up('md')]: {
+      paddingLeft: '35%',
+      paddingRight: '35%',
+    },
+  },
+  title: {
+    fontSize: 48,
+    marginBottom: theme.spacing(4)
+  },
+  rowBox: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'space-between',
```

Comment:
Why not to establish a global theme and have everything in common in just one place?. Instead of writing specific styles for each component. You know what I mean?

Reply:
Yes, thats the goal, the problem is that there is a lot a code that we need to check. So in order for me to be more accurate and organize we are dividing this work in 3 steps. 
1. Add specific styles for each component since there are components that need that. (Routes) and remove duplicated code
2. Add specific styles for each component since there are components that need that. (Components) and remove duplicated code
3. Standarice certain styles for the whole app. Like the example you are pointing out

Reply:
Ok, great!